libraries/networking/src/udt/Connection.cpp

```diff
@@ -0,0 +1,597 @@
+//
+//  Connection.cpp
+//  libraries/networking/src/udt
+//
+//  Created by Clement on 7/27/15.
+//  Copyright 2015 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+#include "Connection.h"
+
+#include <QtCore/QThread>
+
+#include <NumericalConstants.h>
+
+#include "../HifiSockAddr.h"
+#include "CongestionControl.h"
+#include "ControlPacket.h"
+#include "Packet.h"
+#include "Socket.h"
+
+using namespace udt;
+using namespace std;
+using namespace std::chrono;
```

Comment:
I'm not personally a fan of this... It's more verbose to write `std::...`, but not that much more typing, and adds clarity to the code itself.